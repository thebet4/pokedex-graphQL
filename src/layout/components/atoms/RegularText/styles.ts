import { colors } from '@config/theme'
import styled from 'styled-components/native'

export const Regular = styled.Text`
  color: ${colors.purple[200]}
  font-family: "Inter_400Regular";
  font-size: 14px;
  line-height: 17px;
`
