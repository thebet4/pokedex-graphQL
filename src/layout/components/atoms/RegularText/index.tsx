import React, { ReactNode } from 'react'
import { TextStyle } from 'react-native'
import { Regular } from './styles'

interface ITitleProps {
  children: ReactNode
  props?: TextStyle
}

const RegularText = ({ children, props }: ITitleProps) => {
  return <Regular {...props}>{children}</Regular>
}

export default RegularText
