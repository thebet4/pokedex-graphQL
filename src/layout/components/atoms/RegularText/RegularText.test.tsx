import * as React from 'react'
import renderer from 'react-test-renderer'

import RegularText from '@layout/components/atoms/RegularText'

it(`renders <RegularText /> correctly`, () => {
  const tree = renderer.create(<RegularText>Pokemon</RegularText>).toJSON()

  expect(tree).toMatchSnapshot()
})
