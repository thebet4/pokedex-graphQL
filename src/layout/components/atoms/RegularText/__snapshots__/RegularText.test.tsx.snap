// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <RegularText /> correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#21386E",
        "fontFamily": "Inter_400Regular",
        "fontSize": 14,
        "lineHeight": 17,
      },
    ]
  }
>
  Pokemon
</Text>
`;
